cluster/ci/config/prow/config.yaml

```diff
@@ -2087,14 +2087,64 @@ presubmits:
         image: prow-test:latest
 
   kubernetes-incubator/descheduler:
-  - name: ci-kubernetes-descheduler-e2e-gce
-    agent: jenkins
-    labels:
-      master: ci.openshift.redhat.com
+  - name: pull-ci-kubernetes-incubator-descheduler-e2e-gce
+    agent: kubernetes
     always_run: true
-    context: ci/openshift-jenkins
+    context: ci/prow/e2e
+    branches:
+    - master
     rerun_command: "/test e2e"
     trigger: "((?m)^/test( all| e2e),?(\\s+|$))"
+    decorate: true
+    skip_cloning: true
+    spec:
+      serviceAccountName: ci-operator
+      volumes:
+      - name: job-definition
+        configMap:
+          name: prow-job-cluster-launch-src
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-gcp
+          - configMap:
+              name: cluster-profile-gcp
+      containers:
+      - name: test
+        image: ci-operator:latest
+        volumeMounts:
+        - name: job-definition
+          mountPath: /usr/local/e2e-gcp
+          subPath: cluster-launch-src.yaml
+        - name: cluster-profile
+          mountPath: /usr/local/kubernetes-descheduler-e2e-cluster-profile
+        env:
+        - name: JOB_NAME_SAFE
+          value: kubernetes-descheduler-e2e
+        - name: CLUSTER_TYPE
+          value: gcp
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              name: ci-operator-kubernetes-descheduler
+              key: master.json
+        - name: TEST_COMMAND
+          value: "cp /tmp/admin.kubeconfig /tmp/admin.conf && make test-e2e"
+        - name: RPM_REPO_BASEURL_REF
+          value: https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/master/.latest-rpms
```

Comment:
We still need to decide which version to use for the cluster.

Reply:
or do we just want kube?

@smarterclayton